### DISCUSSION TITLE
What about scraping Google Ads?

### DISCUSSION THREAD

**Discussion by s3w47m88** [towfiqi/serpbear#156] [General]
What would be involved to scrape for Google Ads instead of just organic rankings?

**Comment by s3w47m88** [comment]
Bump. If there are no plans for this, I would be happy to have my staff contribute this feature.

**Comment by towfiqi** [comment]
Sorry for the late reply. The feature you requested actually lies outside the intended scope of Serpbear's primary objective, which is to evolve into a comprehensive SEO app.

**Comment by s3w47m88** [comment]
No worries on the response time. Okay, thank you!

For those finding this thread, my staff and I will develop this feature in a fork. If @towfiqi allows it I may post the fork link here when we're done with that.

**Reply by towfiqi** [reply]
Of course! You can post it here.

**Reply by s3w47m88** [reply]
Could you share any considerations for me and my team as we develop this? We're thinking of introducing it as a sort of Plugin so it remains separate from Core and we don't have to fork the entire project. Thoughts?
